snare.py

```diff
@@ -79,6 +79,28 @@ def submit_slurp(self, data):
         except Exception as e:
             print(e)
 
+    def create_data(self, request):
+        data = dict(
```

Comment:
Do you think it would make sense to add the HTTP response status? Would make it easier for us to see if we were able to handle the request.